marquez_airflow/dag.py

```diff
@@ -11,36 +11,36 @@
 # limitations under the License.
 
 import json
+import logging
```

Comment:
Mind if we introduce the logging change in a separate PR to allow for a quick merge? I'd be great if we could isolate the change given logging isn't currently working.

Reply:
not sure, why it wasn't there before. without it no log shows up for me.
but, i reverted other log call rename changes to back to self.log.
no harm keeping the import line ? thanks.

Reply:
That's cool 👍 My suggestion was we'd open a separate PR that had the logging fix in order to isolate the change.